src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -4966,11 +4908,11 @@ namespace EvaporativeCoolers {
         }
 
         if (ZoneEvapUnit(UnitNum).EvapCooler_1_AvailStatus) {
-            SimEvapCooler(ZoneEvapUnit(UnitNum).EvapCooler_1_Name, ZoneEvapUnit(UnitNum).EvapCooler_1_Index);
+            SimEvapCooler(ZoneEvapUnit(UnitNum).EvapCooler_1_Name, ZoneEvapUnit(UnitNum).EvapCooler_1_Index, FanSpeedRatio);
```

Comment:
As @mjwitte found with a lower max TU flow rate than the branch max flow, this isn't really the PLR of the evap cooler. I assume the diff's found yesterday would still be there.

Reply:
PLR is passed to zone evaporative coolers only. When an evap cooler in airloop the PLR defaults to 1.0. The fanspeedratio is used when the zone HVAC evaporative cooler control type is ZoneCoolingLoadVariableSpeedFan.

Reply:
Ok then, as long as this is the actual PLR based on time.